src/displayapp/screens/StopWatch.cpp

```diff
@@ -4,37 +4,40 @@
 #include "displayapp/InfiniTimeTheme.h"
 
 using namespace Pinetime::Applications::Screens;
+using namespace Pinetime::Controllers;
 
 namespace {
-  TimeSeparated_t convertTicksToTimeSegments(const TickType_t timeElapsed) {
+  TimeSeparated convertTicksToTimeSegments(const TickType_t timeElapsed) {
```

Comment:
Since you're refactoring already, could you fix the names of these functions here to be PascalCase as per the style guidelines?

Reply:
Done.